recipes/cublasmp/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: cublasmp
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cublasmp/redist/libcublasmp/LICENSE.txt
+  sha256: c2a95a3d181879325e56333d7a592bb6ba5ab3a44bfde9574513ccc2f87f9c68
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
Remind me, can we build `noarch` packages if the dependencies are not `noarch`?

Reply:
Yes we do for cuda. `cuda-toolkit` is noarch, and dependencies are not https://github.com/conda-forge/cuda-toolkit-feedstock/blob/main/recipe/meta.yaml#L13

We've only ran into trouble when we tried to do arch specific selectors in a noarch package.